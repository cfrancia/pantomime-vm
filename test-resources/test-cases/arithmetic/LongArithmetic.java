/**
 * Created by cfrancia on 24/09/16.
 */
public class LongArithmetic {

    public static void main(String[] args) {
        add(100L, 5L);
        subtract(100L, 5L);
        multiply(100L, 5L);
        divide(100L, 5L);
    }

    public static void add(long a, long b) {
        println(a + b);
    }

    public static void subtract(long a, long b) {
        println(a - b);
    }

    public static void multiply(long a, long b) {
        println(a * b);
    }

    public static void divide(long a, long b) {
        println(a / b);
    }

    public static native void println(long val);

}
