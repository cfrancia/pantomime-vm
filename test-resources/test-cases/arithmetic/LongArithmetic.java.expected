OUT: 105
OUT: 95
OUT: 500
OUT: 20
